theano/gpuarray/tests/test_pickle.py

```diff
@@ -29,7 +29,7 @@
         cuda_ndarray.use('gpu', default_to_move_computation_to_gpu=False,
                          move_shared_float32_to_gpu=False,
                          enable_cuda=False)
-        theano.sandbox.gpuarray.init_dev('cuda')
```

Comment:
I'm ok with that.

Though I suspect that this code is why we saw the testsuite use two GPUs for no reason.